tests/util/blockchain.py

```diff
@@ -35,9 +35,11 @@ async def create_blockchain(constants: ConsensusConstants, db_version: int):
     return bc1, connection, db_path
 
 
+# @pytest.fixture(scope="module")
```

Comment:
these comments should be removed before we land (and maybe experimented with further in a separate PR)